Fixed #24895 -- Fixed loading a pair of squashed migrations with a dependency.

carljm opened the pull request.

carljm commented:
buildbot, add to whitelist.

charettes commented on tests/migrations/test_loader.py:
```diff
@@ -267,6 +267,32 @@ def test_loading_squashed_complex_multi_apps(self):
+            ('app1', '4_auto'),
+            ('app1', '3_auto'),
+            ('app2', '1_squashed_2'),
+        ])
```
You could use a set literal here.

carljm replied:
True. I was just being consistent with neighboring similar tests, which don't.

charettes replied:
Ahh didn't notice that. I just remembered a recent commit that converted many of those to literals.

timgraham commented:
Looks good besides more blank lines in the test than I'd opt for and missing release notes if you plan to backport to 1.8.

carljm commented:
Thanks for reviews! Removed a blank line in the test, converted the new test and the one right before it to use set literals (the only tests in this file that needed that conversion), and added release note for 1.8.3. Will merge when CI is green.
